libs/wire-api/src/Wire/API/Arbitrary.hs

```diff
@@ -1,6 +1,5 @@
-{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE DisambiguateRecordFields #-}
```

Comment:
oops?  why did you turn this on if it obviously compiled before?

Reply:
ah, probably some overlap with RecordWildCards?

Reply:
Indeed, `DisambiguateRecordFields` is implied by `RecordWildCards`, but `DisambiguateRecordFields` is sufficient here.

Probably, this doesn't buy us much, but reducing HLint warnings helps to make the more important ones easier to spot...